docs/alexa-event.rst

```diff
@@ -7,24 +7,28 @@ The ``AlexaEvent`` Object
 
   The ``alexaEvent`` object contains all the information from the Voxa event, it's an object kept for the entire lifecycle of the state machine transitions and as such is a perfect place for middleware to put information that should be available on every request.
 
-  .. js:attribute:: AlexaEvent.intent
 
-    In Voxa the alexaEvent object makes ``intent.slots`` available through ``intent.params`` after aplying a simple transformation so ``{ slots: [{ name: 'Dish', value: 'Fried Chicken' }] }`` becomes ``{ Dish: 'Fried Chicken' }``, in other platforms it does it best to make the intent params for each platform also available on ``intent.params``
+  .. js:attribute:: AlexaEvent.token
 
-  .. js:attribute:: AlexaEvent.user
+    A convenience getter to obtain the request's token, specially when using the ``Display.ElementSelected``
 
-    A convenience getter to obtain the user from ``session.user`` or ``context.System.user`` in alexa, and ``conv.user.id`` in dialogflow. In other platforms it's also available, you can always count on the ``alexaEvent.user.userId`` being available. If there's an ``accessToken`` it will also be available through ``alexaEvent.user.accessToken``
+  .. js:attribute:: AlexaEvent.alexa.customerContact
 
-  .. js:attribute:: AlexaEvent.token
 
-    A convenience getter to obtain the request's token, specially when using the ``Display.ElementSelected``
+    When a customer enables your Alexa skill, your skill can request the customer's permission to the their contact information, see :ref:`alexa-customer-contact`.
+
+  .. js:attribute:: AlexaEvent.alexa.deviceAddress
+
+    When a customer enables your Alexa skill, your skill can obtain the customer's permission to use address data associated with the customer's Alexa device., see :ref:`alexa-device-address`.
+
+  .. js:attribute:: AlexaEvent.alexa.deviceSettings
 
-  .. js:attribute:: AlexaEvent.requestToIntent
```

Comment:
i removed this cause it's a method meant to be used internally by the framework. I also updated the code to have this as a protected method